engine/runtime/src/main/java/org/enso/interpreter/Language.java

```diff
@@ -60,11 +63,18 @@
    */
   @Override
   protected Context createContext(Env env) {
-    Context context = new Context(this, getLanguageHome(), env);
+    var notificationHandler = new Forwarder();
+    boolean isInteractiveMode = env.getOptions().get(RuntimeOptions.INTERACTIVE_MODE_KEY);
+    boolean isTextMode = !isInteractiveMode;
+    if (!isTextMode) {
```

Comment:
are you just inverted interactive mode twice, or I'm missing something?

Reply:
Oh that is a mistake, it was supposed to be `if(isTextMode)`! Thanks!